src/MTransformationMatrix.inl

```diff
@@ -0,0 +1,322 @@
+
+#define _doc_TransformationMatrix_reorderRotation \
+    "Reorders the transformation's rotate component to give the same overall rotation but using a new order or rotations."
+
+#define _doc_TransformationMatrix_setToRotationAxis \
+    "Sets the transformation's rotate component to be a given axis vector and angle in radians."
+
+#define _doc_TransformationMatrix_rotationOrientation \
+    "Returns a quaternion which orients the local rotation space."
+
+#define _doc_TransformationMatrix_setRotationOrientation \
+    "Sets a quaternion which orients the local rotation space."
+
+#define _doc_TransformationMatrix_setTranslation \
+    "Sets the transformation's translation component."
+
+#define _doc_TransformationMatrix_setShear \
+    "Sets the transformation's shear component."
+
+#define _doc_TransformationMatrix_scalePivot \
+    "Returns the transformation's scale pivot component."
+
+#define _doc_TransformationMatrix_setScalePivot \
+    "Sets the transformation's scale pivot component."
+
+#define _doc_TransformationMatrix_scalePivotTranslation \
+    "Returns the transformation's scale pivot translation component."
+
+#define _doc_TransformationMatrix_setScalePivotTranslation \
+    "Sets the transformation's scale pivot translation component."
+
+#define _doc_TransformationMatrix_rotatePivot \
+    "Returns the transformation's rotate pivot component."
+
+#define _doc_TransformationMatrix_setRotatePivot \
+    "Sets the transformation's rotate pivot component."
+
+#define _doc_TransformationMatrix_rotatePivotTranslation \
+    "Returns the transformation's rotate pivot translation component."
+
+#define _doc_TransformationMatrix_setRotatePivotTranslation \
+    "Sets the transformation's rotate pivot translation component."
+
+#define _doc_TransformationMatrix_isEquivalent \
+    "Returns true if this transformation's matrix is within tolerance of another's matrix."
+
+py::enum_<MTransformationMatrix::RotationOrder>(TransformationMatrix, "RotationOrder")
+    .value("kInvalid", MTransformationMatrix::RotationOrder::kInvalid) 
+    .value("kXYZ", MTransformationMatrix::RotationOrder::kXYZ)
+    .value("kYZX", MTransformationMatrix::RotationOrder::kYZX)
+    .value("kZXY", MTransformationMatrix::RotationOrder::kZXY)
+    .value("kXZY", MTransformationMatrix::RotationOrder::kXZY)
+    .value("kYXZ", MTransformationMatrix::RotationOrder::kYXZ)
+    .value("kZYX", MTransformationMatrix::RotationOrder::kZYX)
+    .value("kLast", MTransformationMatrix::RotationOrder::kLast)
+    .export_values();
+
+TransformationMatrix
+    .def(py::init<>())
+    .def(py::init<const MMatrix &>(), py::arg("src"))
+    .def(py::init<const MTransformationMatrix &>(), py::arg("src"))
+
+    .def("asMatrix", [](MTransformationMatrix & self, double interp = 1.0) -> MMatrix {
+        return self.asMatrix(interp);
+    }, _doc_TransformationMatrix_asMatrix)
```

Comment:
You should use `py::arg` in order for the stubs to get the correct name and value for the `interp` argument.

```suggestion
    }, py::arg("interp") = 1.0, _doc_TransformationMatrix_asMatrix)
```